.github/label-change.yml

```diff
@@ -31,3 +31,28 @@ ci:
   - ".github/**"
   - "dockers/**"
   - ".pre-commit-config.yml"
+
+docs:
+  - ".github/workflows/docs-build.yml"
+  - "docs/**"
```

Comment:
I would not add the docs label to all PR that touch this path, as many PRs make slight modifications to the docs but they are not generally considered a "docs PR"

```suggestion
```

Reply:
but still, yu may need to check the final docs, the same way you can say about app that a minor style change can add the label without functional change